website/versioned_docs/version-1.0/appd-spec.md

```diff
@@ -5,7 +5,7 @@ original_id: appd-spec
 ---
 ## API
 
-View the full specification in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://rebilly.github.io/ReDoc/)).
+View the [full specification](/1.0/appd-spec) in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://rebilly.github.io/ReDoc/)).
```

Comment:
/docs/1.0/appd-spec ?

Reply:
Nope, that the page on which the link exists, trying to find what the link to the redoc page is... appears to have changed in this PR (both the link on the page and the correct link to the file, chance might need a fix here)

Reply:
```suggestion
View the [full specification](/schemas/1.0/app-directory) in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://rebilly.github.io/ReDoc/)).
```

This'll fix it, looks like you moved the page to match the other versions, so link changed